libs/language-server/src/lib/ast/expressions/evaluators/parse-operators-evaluators.spec.ts

```diff
@@ -0,0 +1,37 @@
+// SPDX-FileCopyrightText: 2024 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+import { executeDefaultTextToTextExpression } from '../test-utils';
+
+describe('The asDecimal operator', () => {
+  it('should parse decimals successfully', async () => {
+    const result = await executeDefaultTextToTextExpression(
+      'asDecimal inputValue',
+      '1.0',
+    );
+
+    expect(result).toEqual(1);
+  });
+});
+
+describe('The asInteger operator', () => {
+  it('should parse integers successfully', async () => {
+    const result = await executeDefaultTextToTextExpression(
+      'asInteger inputValue',
+      '32',
+    );
+
+    expect(result).toEqual(32);
+  });
+});
+
+describe('The asBoolean operator', () => {
+  it('should parse integers successfully', async () => {
```

Comment:
A few more values would be great, e.g. 32.5 for integer, -1 for integer, '0' for boolean etc.